docs/source/nodes_and_pipelines/namespaces.md

```diff
@@ -0,0 +1,336 @@
+# Reuse pipelines with namespaces
+
+## How to reuse your pipelines
+
+If you want to create a new pipeline that performs similar tasks with different inputs/outputs/parameters as your `existing_pipeline`, you can use the same `pipeline()` creation function as described in [How to structure your pipeline creation](modular_pipelines.md#how-to-structure-your-pipeline-creation). This function allows you to overwrite inputs, outputs, and parameters. Your new pipeline creation code should look like this:
+
+```python
+def create_new_pipeline(**kwargs) -> Pipeline:
```

Comment:
I think this PR overall uses less specific names than the current docs, which at least to me is an impact on docs quality. Compare:

`create_new_pipeline`

to:

```python
final_pipeline = (
    cook_breakfast_pipeline
    + eat_breakfast_pipeline
    + cook_lunch_pipeline
    + eat_lunch_pipeline
)
```

Reply:
I updated it to` create_pipeline()`. If I understood correctly, the name `create_pipeline()` is important (although it is not specific) because otherwise, the pipeline will not be autodiscovered and executed during `kedro new` (see [Kedro documentation](https://docs.kedro.org/en/stable/nodes_and_pipelines/pipeline_registry.html)). If you see any other examples with less specific names, please let me know and I would be happy to fix them as well.